WikiWatershed/model-my-watershed
# Revamp Sidebars + Style Touchups

## Overview

This PR began as a humble attempt at matching the analyze stage to the new wireframes (#1871). The existing styling applied to the sidebar, however, was so heavily nested, it was difficult move even simple pieces of text in the view. I decided to wrap #1864 in, and try to refactor the sidebar-related stylesheets. I then realized whatever I merged would possibly redo parts of the work of #1844, and definitely create even more merge conflicts for it. @jfrankl and I paired, and to merge my work, the changes on develop, and #1844.

**Wireframes**
https://marvelapp.com/gji4cg6/screen/27712989
<details>

<summary>
Here's a selection of them
</summary>

![screen shot 2017-05-15 at 4 15 37 pm](https://cloud.githubusercontent.com/assets/7633670/26077382/dbca2f42-3989-11e7-956e-7aaca029e9d4.png)


![screen shot 2017-05-15 at 4 15 44 pm](https://cloud.githubusercontent.com/assets/7633670/26077388/df1c99aa-3989-11e7-8e24-e490a1f2bff3.png)


![screen shot 2017-05-15 at 4 15 51 pm](https://cloud.githubusercontent.com/assets/7633670/26077393/e27394b4-3989-11e7-978e-947ff74aba67.png)

![screen shot 2017-05-15 at 4 15 57 pm](https://cloud.githubusercontent.com/assets/7633670/26077404/e6cbe5e8-3989-11e7-9cc6-51cf92d8cf74.png)


</details>



Connects #1844 
Connects #1864 
Connects #1871 

### Demo

![screen shot 2017-05-15 at 3 53 48 pm](https://cloud.githubusercontent.com/assets/7633670/26077428/faf6d85c-3989-11e7-98db-d388267438c5.png)

![screen shot 2017-05-15 at 3 56 22 pm](https://cloud.githubusercontent.com/assets/7633670/26077426/faf555ae-3989-11e7-8c15-aa0d79c55f6e.png)

<img width="231" alt="screen shot 2017-05-15 at 11 20 15 am" src="https://cloud.githubusercontent.com/assets/7633670/26077576/8c74250a-398a-11e7-95c0-17b597789721.png">


<img width="335" alt="screen shot 2017-05-15 at 11 20 39 am" src="https://cloud.githubusercontent.com/assets/7633670/26077567/88546d22-398a-11e7-8a1b-a7442633fc00.png">

![screen shot 2017-05-15 at 3 57 54 pm](https://cloud.githubusercontent.com/assets/7633670/26077425/faf4aac8-3989-11e7-9e98-5fef0e6b2795.png)

![screen shot 2017-05-15 at 3 57 58 pm](https://cloud.githubusercontent.com/assets/7633670/26077429/fafa03ba-3989-11e7-8908-070fa44b7930.png)

![screen shot 2017-05-15 at 3 58 09 pm](https://cloud.githubusercontent.com/assets/7633670/26077424/faf44d6c-3989-11e7-98e3-270d99e08215.png)


### Notes

In the wireframes, when the analyze tab is visible in the model stage, we hide the project toolbar, and show the AOI header. I'll be spinning this work off into another card.

![screen shot 2017-05-15 at 4 16 06 pm](https://cloud.githubusercontent.com/assets/7633670/26077414/edff84dc-3989-11e7-9e56-dfc95fd27f94.png)

## Testing Instructions

 * Pull branch, bundle
In MMW mode
 * Move through the app, splash to model, at various viewport sizes, devices, and browsers
 * Ensure everything but the app header, scenario toolbar, project header, and splash is approaching the wireframe styling

In BigCZ mode, check that the data source button looks okay and takes you to the datasource search sidebar
